### PR TITLE
feat: add layout

### PR DESCRIPTION
**Pull request by JacksonTian**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by JacksonTian** [comment]
得去改egg-mock啊～～～

File: lib/view.js [hunk 1]
```diff
@@ -24,6 +22,18 @@ module.exports = class EjsView {
     });
   }
 
+  * render(filename, locals, viewOptions) {
+    const config = Object.assign({}, this.config, viewOptions, { filename });
+    const html = yield this._render(filename, locals, config);
+    if (config.layout) {
```
Comment (popomore) [inline lib/view.js]: 反着判断吧，尽量都写成一行

```
if (!config.layout) return html;

```

File: lib/view.js [hunk 2]
```diff
@@ -10,9 +10,7 @@ module.exports = class EjsView {
     this.config = ctx.app.config.ejs;
   }
 
-  render(filename, locals, viewOptions) {
-    const config = Object.assign({}, this.config, viewOptions, { filename });
-
+  _render(filename, locals, config) {
```
Comment (popomore) [inline lib/view.js]: 用 symbol

**Review by popomore — CHANGES_REQUESTED** [review]
加个用例吧

**Comment by popomore** [comment]
> 得去改egg-mock啊

这个啥意思

**Review by popomore — APPROVED** [review]

**Comment by popomore** [comment]
+ egg-view-ejs@1.1.0

**Pull request merged**